### PR TITLE
Message linking client server

### PR DESCRIPTION
**Pull request by FlorisXll**
From the server point of view:

- made an EvenCompletedMessage
- registered that message
- interpeted that message

Client: https://github.com/jessetilro/pixelperfect-client/pull/7
Issue: #95

### CODE REVIEW

**Comment by FlorisXll** [comment]
I will need to resolve conflicts first.

**Comment by dmalarev** [comment]
Code looks good so far

**Comment by jessetilro** [comment]
Checked out both branches and did a manual end-to-end test, works nicely. Next step forward would be to have the button presses solve a specific event, am I right?

File: src/main/java/nl/tudelft/pixelperfect/client/ServerListener.java [hunk 1]
```diff
@@ -51,6 +51,9 @@ public void messageReceived(HostedConnection source, Message message) {
       HelloMessage helloMessage = (HelloMessage) message;
```
Comment (jessetilro) [inline src/main/java/nl/tudelft/pixelperfect/client/ServerListener.java]: The HelloMessage class can be removed.

**Comment by jessetilro** [comment]
I am missing a test suite for the EventCompletedMessage, is this class temporary?

File: src/main/java/nl/tudelft/pixelperfect/client/ServerListener.java (hunk 1 above)
Comment (FlorisXll) [inline src/main/java/nl/tudelft/pixelperfect/client/ServerListener.java]: Alright, will resolve this in my next Issue.

**Comment by FlorisXll** [comment]
Next PR: fixing Static Erros/Tests.

Next step/Issue: Implenting specific buttons for each event that arrives at the client. Each button will complete the event.

#74 #95 #119

**Comment by dmalarev** [comment]
So, can this PR be merged or is there something that needs to be done?

**Comment by jessetilro** [comment]
Alright I approve, but next time the tests should really be updated in the same PR.

**Comment by dalderliesten** [comment]
Looks good to me for now, but please do add tests next time.  Otherwise no remarks!

**Pull request merged**